pkg/pxc/backup/restore.go

```diff
@@ -173,8 +178,13 @@ func RestoreJob(cr *api.PerconaXtraDBClusterRestore, bcp *api.PerconaXtraDBClust
 		},
 		app.GetSecretVolumes("vault-keyring-secret", cluster.Spec.PXC.VaultSecretName, true),
 	}
-	var command []string
 
+	sslVolume := app.GetSecretVolumes("ssl", cluster.Spec.PXC.SSLSecretName, cluster.Spec.AllowUnsafeConfig)
+	if cluster.CompareVersionWith("1.15.0") >= 0 {
```

Comment:
Maybe to swap these two like:
Always run `sslVolume = app.GetSecretVolumes("ssl", cluster.Spec.PXC.SSLSecretName, !cluster.TLSEnabled())` but if the version is below 1.15.0 do the other thing.

This way the new thing is the default, it is easier to deprecate the old one and for 1.15.0 clusters we don't need to run both things always.

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/1744/commits/e5380e648441e51dea2634ba3475a2ee39fa1fdb